src/test/regress/specs/isolation_master_append_table.spec

```diff
@@ -0,0 +1,97 @@
+setup
+{	
+  	CREATE TABLE table_to_append(id int);
+  	CREATE TABLE table_to_be_appended(id int);
+  	CREATE TABLE table_to_distribute(id int);
+
+  	SELECT create_distributed_table('table_to_append', 'id', 'append');
+  	INSERT INTO table_to_be_appended VALUES(1),(2),(3),(4),(5),(6);
+  	INSERT INTO table_to_distribute SELECT generate_series(1,100);
```

Comment:
Well, I think we should move the unrelated tests to separate files and give names accordingly. That's important for several reasons (i) Run them in parallel (yes, let's run those files in parallel:) ) (ii) Expand the tests in the future (which is very likely, I already suggested two new tests in this PR) (iii) The existing tests follow that pattern

So,

file1: isolation_master_append_table
file2: isolation_create_distributed_table
file3: isolation_master_apply_delete

Does that sound good to you as well?

Reply:
Yes, I was actually thinking about doing the same. I will update accordingly

Reply:
>file2: isolation_create_distributed_table

So, for example, we could add concurrent COPY to local table and create_distributed table. Which should be a valuable test

Reply:
Do you have any suggestion on putting the concurrent apply_delete vs. append table tests under which file?

Reply:
nvm, I am removing these tests.